data/korath/korath missions.txt

```diff
@@ -257,3 +257,566 @@ mission "Small Scale Delivery"
+			`	Their companion picks up the story. "Vas, certu vas. Minli ka Korath torq. Kem'lor ka forli, ka merru, ka plee. Teka korma ka draki, rekki korath, marre fa long." They draw a bunch of small spiders, surrounding the Korath ships, then gathers the sand from that part of the map and tosses it onto the place where the first Korath ship was drawn. "Tarre ta fo'lu korath."`
+			`	The other one says, "Far'en lai, soru ka'lai en ka korath. Soti far'en lai ko forei lak'far." It points to the gun at the front of its fighter, then makes a gesture, hands turned downward toward the ground. "Far'en lai ka let'mari nik lak'mal." It points to a few places, hundreds of meters away, where you see green plants growing. "Nik lak'mal." Then it points to you. "Hoomani ka Korath nik lak'mal."`
+			`	To emphasize the point, on a blank part of the sand they draw a circle, very big, with lots of dots on it, pointing to the plants as they make each dot. They then draw on outline of all three of your ships inside the circle, pointing towards each other. Finally, one picks up the sand they just used to make your ship, and tosses it back onto the map of human space, while the other picks up the sand that makes up their ships, and likewise tosses it onto the map of their own space.`
+			choice
+				`	(Try to assure them you do not mean to fight them.)`
+					goto assurance
+				`	(Try to communicate that they should leave humanity alone.)`
+					goto alone
+
+			label assurance
+			`	"We do not want to fight you," you say, and you redraw the human and Korath ships, on the map, so they are facing away from each other.`
+			`	"Kai," says one of the Korath, looking at the other, "mik'mal ka puk?"`
+			`	"Lajalness?" the other says back, and takes a moment to look at your ship.`
+			`	Returning to the sand, they make another copy of your respective ships, this time, all facing in the same direction. Making a gesture, they alter the drawing you made, so that the ships appear double-ended. They point to each drawing, and then to you.`
+			choice
+				`	(Point to the three ships flying together in the same direction.)`
+					goto together
+				`	(Point to the double-ended ships facing off against each other.)`
+					goto fight
+				`	(Change the drawing to emphasize the ships moving apart again.)`
+					goto alone
+
+			label together
+			`	They stare at you for another few awkward seconds.`
+			`	Then, in front of your three ships, they slowly and carefully draw a large fire, larger than the three of your ships put together. Beside your ships, facing away, they draw a pointed circle on one side, and a spider on the other; after a brief pause, they make the circle and spider double-ended. Again, they point to the drawing of your double-ended ships, and then to the now ominous-looking collection of ships and fire, and point to you.`
```

Comment:
It's a "pointed circle," so it does have a direction. Again, I will ping @ravenshining for comment.

Reply:
If I understand correctly, this would be the best change:

Beside your ships, facing away, they draw a pointed circle on one side, and a spider on the other. **After a brief pause, they give the spider heads on both sides of its body, and give the circle two points.** Again, they point to the drawing of your double-ended ships, and then to the now ominous-looking collection of ships and fire, and point to you.

Reply:
A raven whispered to me, and I have these changes:

> Then, in front of your three ships, they slowly and carefully draw a large fire, larger than the three of your ships put together. **Beside your ships, they draw a teardrop shape on one side, facing away. On the other side, they draw a spider facing in the opposite direction. After a brief pause, they give the spider heads on both sides of its body, and give the teardrop a second tail.** Again, they point to the drawing of your double-ended ships, and then to the now ominous-looking collection of ships and fire, and point to you

The "pointed circle" was a teardrop, and the one with two points is a teardrop with two tails.